src/plugins/vis_type_vega/public/vega_view/vega_base_view.js

```diff
@@ -189,7 +189,11 @@ export class VegaBaseView {
           })
         );
       }
-      return originalSanitize(uri, options);
+      const result = await originalSanitize(uri, options);
+      // This will allow Vega users to load images from any domain.
+      result.crossOrigin = null;
```

Comment:
@wylieconlon I'm not a big fan of this kind of changes.  Could you please help me to reproduce that issue locally. I've modified `hosts` file, set `vis_type_vega.enableExternalUrls: true` and use the following vega spec with image from external resource. 

Vega spec: 
```
{
  $schema: https://vega.github.io/schema/vega/v5.json
  marks: [
  {
      "type": "image",
      "encode": {
        "enter": {
          "url": {
            "value": "https://cdn.glitch.com/4c9ebeb9-8b9a-4adc-ad0a-238d9ae00bb5%2Fmdn_logo-only_color.svg?1535749917189"
          }
        }
      }
    }
  
  ]
}
```

Result: 
![image](https://user-images.githubusercontent.com/20072247/108506338-a52b3f80-72c9-11eb-8e69-aa258afd7282.png)

Image loaded without any warnings/errors in console

Reply:
@alexwizp the image that you're found allows all CORS requests, so it is not a good example of the type of image that causes this behavior. Most CDNs automatically allows CORS requests for images, so maybe the best place to find this kind of image is a non-CDN source. Maybe setting up a local HTTP server on a different port would be a better example.